aiosmtpd/smtp.py

```diff
@@ -20,6 +20,8 @@
 EMPTYBYTES = b''
 NEWLINE = '\n'
 MISSING = object()
+XCLIENT_ATTRIBUTES = (
+    'ADDR', 'DESTADDR', 'DESTPORT', 'HELO', 'LOGIN', 'NAME', 'PORT', 'PROTO')
```

Comment:
I'm -1 on putting these set of attributes on the module level.

It should be within the class itself, as a class attribute or within the `smtp_XCLIENT` method.

Reply:
On second thought, it should be a class / instance attribute.